src/test/java/io/xpring/xrpl/XrpAccountDeleteProtoConversionTest.java

```diff
@@ -0,0 +1,53 @@
+package io.xpring.xrpl;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+import io.xpring.common.XrplNetwork;
+import io.xpring.xrpl.model.XrpAccountDelete;
+import org.junit.Test;
+import org.xrpl.rpc.v1.AccountDelete;
+
+public class XrpAccountDeleteProtoConversionTest {
+  @Test
+  public void allFieldsSetTest() {
+    // GIVEN an AccountDelete protocol buffer with all fields set.
+    final AccountDelete accountDeleteProto = FakeXrpTransactionProtobufs.allFieldsAccountDelete;
+
+    // WHEN the protocol buffer is converted to a native Java object.
+    final XrpAccountDelete accountDelete = XrpAccountDelete.from(accountDeleteProto, XrplNetwork.TEST);
+
+    ClassicAddress classicAddress = ImmutableClassicAddress.builder()
+      .address(accountDeleteProto.getDestination().getValue().getAddress())
+      .tag(accountDeleteProto.getDestinationTag().getValue())
+      .isTest(true)
+      .build();
+    final String destinationXAddress = Utils.encodeXAddress(classicAddress);
+
+    // THEN the AccountDelete converted as expected.
+    assertThat(accountDelete.destinationXAddress()).isEqualTo(destinationXAddress);
+  }
+
+  @Test
+  public void destinationTagNotSetTest() {
+    // GIVEN an AccountDelete protocol buffer with no destination tag.
+    final AccountDelete accountDeleteProto = FakeXrpTransactionProtobufs.noDestinationTagAccountDelete;
+
+    // WHEN the protocol buffer is converted to a native Java object.
+    final XrpAccountDelete accountDelete = XrpAccountDelete.from(accountDeleteProto, XrplNetwork.TEST);
+
+    ClassicAddress classicAddress = ImmutableClassicAddress.builder()
+      .address(accountDeleteProto.getDestination().getValue().getAddress())
+      .isTest(true)
```

Comment:
i purposefully didn't use the `destinationTag` function in the builder. should i do this? when i did, it gave a different result, presumably because the default value for `Integer` is 0. not sure how these are used in practice

Reply:
Yep, I think that's the right thing to do